packages/components/menu/src/menu.tsx

```diff
@@ -47,11 +47,12 @@ function Menu<T extends object>(props: Props<T>, ref: ForwardedRef<HTMLUListElem
           hideSelectedIcon,
           ...item.props,
         };
+        const mergedItemClasses = itemProps.classNames ?? itemClasses;
```

Comment:
may double check if the expected behaviour is "merge" or "override". The naming suggest it's "merge" but the logic is "override". If it's the latter, we may also need to point that out in documentation.

Reply:
updated the documentation